src-env (msdb)/_MicroserviceTemplate_.WebApi/Properties/launchSettings.json

```diff
@@ -0,0 +1,89 @@
+{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:57886",
+      "sslPort": 0
+    }
+  },
+  "profiles": {
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "launchUrl": "swagger",
+      "environmentVariables": {
+        "AspNetCore_Environment": "Development",
+        "ConnectionStrings": "Data Source=(localdb)\\mssqllocaldb;Integrated Security=SSPI;Initial Catalog=Benraz._MicroserviceTemplate_",
+        
+        "Audience": "_MicroserviceTemplate_",
+
+        "General_LogFolder": "./Logs",
+        "Logging_LogLevel_Default": "Information",
+        "Logging_LogLevel_Microsoft": "Warning",
+        "Logging_LogLevel_Microsoft.Hosting.Lifetime": "Information",
+
+        "AllowedHosts": "*",
+        "DisableSwagger": "false",
+        "InjectDBCredentialFromEnvironment": "true",
+        "SwaggerBasePath": "_microservicetemplate_-server",
+        "EnablesApplicationInsights": "false",
+
+        "Region": "USE1",
+        "Environment": "DEV",
+        "ServiceName": "_microservicetemplate_-server",
+        "Service": "MSTS",
+
+        "General_AuthorizationBaseUrl": "null",
+        "General_AuthorizationAccessToken": "null",
+        "TokenValidation_Audience": "null",
+
+        "Jwt_Audience": "",
+        "Jwt_Authority": "",
+
+        "AccessToken": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyLCJyb2xlIjoidGVzdCJ9.-I0pOAoVASxAkkupXlue8npTy_LBPua_ln_V6Tq_-_8",
+        "BenrazAuthGateway_BaseUrl": "http://localhost:60341",
+
+        "AspNetCoreDbUserName": "__DB_USERNAME__",
+        "AspNetCoreDbPassword": "__DB_PASSWORD__"
+      }
+    },
+    "_MicroserviceTemplate_.WebApi": {
```

Comment:
why we have 2 env in the launch ?

Reply:
I removed environment variables from the **_MicroserviceTemplate_.WebApi** profile.
While local development we used **IIS Express** profile.